app/views/users/show.html.erb

```diff
@@ -1,11 +1,35 @@
-<%# <p style="color: green"><%= notice</p>
+<p style="color: green"><%= notice %></p>
 
-<%# <%= render @user %> 
+<%= render @user %>
 
-<%# <div>
-  <%# <%= link_to "Edit this user", edit_user_path(@user) %> 
-  <%# <%= link_to "Back to users", users_path %>
+<div class = "container my-5 userBG">
+    <div class = "row">
+      <div class="col-12 row">
+          <h4 class = "col-12 py-2">Bio</h4>
+          <p class = "col-12 py-2 bioText">
+            <%= @user.bio %>
+          </p>
+      </div>
+    </div>
+</div>
```

Comment:
- [ ] it would be better to match the wireframe given  in order to match the requirements 
- wireframe 
<img width="500" alt="Screenshot 2022-12-17 at 13 28 10" src="https://user-images.githubusercontent.com/61790587/208241839-cd087d56-e542-4402-9dcb-2d1947f9d8ea.png">
- yours 
<img width="293" alt="Screenshot 2022-12-17 at 13 24 09" src="https://user-images.githubusercontent.com/61790587/208241842-9e4a691b-1560-4e02-b79b-b917a9649248.png">